pydra/engine/tests/test_submitter.py

```diff
@@ -20,7 +20,7 @@
 from datetime import datetime
 
 slurm_available = bool(shutil.which("sbatch"))
-sge_available = bool(shutil.which("qsub"))
```

Comment:
is this needed within this PR?

Reply:
no, it doesn't have to be in this pr. I was having issue testing on openmind. some nodes had `qsub` command, but not `qacct`

Reply:
the initial PR was meant as a refactor, so don't want to include any breaking changes. In a separate PR, we can probably change `X_available` to check for both: `sacct / sbatch` and `qsub / qacct`

Reply:
yes, I can do it